spec/features/04_user_signs_in_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'spec_helper'
+
+feature "User creates an account", %(
+As a User
+I want to create an account on the Podcast site
+So that I can add and edit reviews (later)
+
+Acceptance Criteria:
+[x] When I go to the user sign-up page, I see sign-up options
+[x] After I sign up, I am redirected to the podcasts index page
+[x] Email must be unique
+[x] Email and password can't be blank
+[] Email must be valid
+[] When I want to sign in, the site recognizes my username and password
+) do
```

Comment:
Let's eliminate these comments. From this point on, we want comments at a bare minimum, if at all. You can use them to create your test, but delete it afterwards.